website/content/docs/nia/configuration.mdx

```diff
@@ -147,7 +147,7 @@ task {
 - `name` - (string: required) Name is the unique name of the task (required). A task name must start with a letter or underscore and may contain only letters, digits, underscores, and dashes.
 - `enabled` - (bool: true) Enable or disable a task from running and managing resources.
 - `providers` - (list[string]) Providers is the list of provider names the task is dependent on. This is used to map [Terraform provider configuration](#terraform-provider) to the task.
-- `services` - (list[string]) Required depending on [`condition`](#condition) configuration. Services is the list of logical service names or service IDs the task executes on. CTS monitors the Consul Catalog for changes to these services and triggers the task to run. Any service value not explicitly defined by a `service` block with a matching ID is assumed to be a logical service name in the default namespace. Alternative to configuring `services`, a `condition` can be configured so that the task does not trigger on changes to services (default behavior) but instead trigger on a different condition. See [Task Condition](#task-condition) configuration for more details.
+- `services` - (list[string]) An optional list of logical service names or service IDs that the task monitors in Consul Catalog for changes. `services` can be configured with a `condition` block of type `consul-kv`, `catalog-services`, or `schedule` to provide additional services source input to the task. Or it can be configured with no `condition` block to set the task's condition so that the task triggers when the services change. Any service value not explicitly defined by a `service` block with a matching ID is assumed to be a logical service name in the default namespace.
```

Comment:
```suggestion
- `services` - (list[string]) An optional list of logical service names or service IDs that the task monitors in Consul Catalog for changes. `services` can be configured with a `condition` block of type `consul-kv`, `catalog-services`, or `schedule` to provide additional services source input to the task. Alternatively, `services` can also be configured with without a `condition` block. Using `services` without a condition block results in a task triggering when the services change. Any service value not explicitly defined by a `service` block with a matching ID, is assumed to be a logical service name in the default namespace.
```
I tried to remove ambiguity. Let me know if the suggestion makes change.

Reply:
I think this adds more clarity, thank you